src/main/Apis/Transactions/TransactionsApi.cs

```diff
@@ -45,5 +45,10 @@ public TransactionInitializeResponse Initialize(TransactionInitializeRequest req
                 "transaction/export",
                 new TransactionExportRequest { From = from, To = to, Settled = settled, PaymentPage = paymentPage }
             );
+
+        public ChargeAuthorizationResponse ChargeAuthorization(ChargeAuthorizationRequest request)
```

Comment:
We can make this method more compact (a one-liner) with the "**expression method**" syntax.

Compilers love them short, too! :)

Reply:
Fixed at [dff94da](https://github.com/adebisi-fa/paystack-dotnet/commit/dff94da738790bb1d04b566e819e980396a55600).